.github/workflows/pull-request.yml

```
@@ -543,4 +543,22 @@ jobs:
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '22.x'
+
+      - name: Install Dependencies
+        run: npm install
+
+      - name: Check for Unused and Missing Dependencies
+        run: npm run check-dependencies
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)